stable/mlrun/values.yaml

```diff
@@ -100,7 +100,7 @@ api:
     httpGet:
       path: /api/healthz
       port: http
-    initialDelaySeconds: 60
+    initialDelaySeconds: 1000
```

Comment:
I guess that's a left over from testing ?
if no explain (in a comment) why so high

Reply:
The test-db you sent me took 7m to migrate from sqlite to mysql. So we need this higher delay, otherwise the pod can be restarted during migration. 
The amount of time is debatable but it should be at least 10m